main/commons/src/main/java/org/cryptomator/common/vaults/WebDavVolume.java

```diff
@@ -98,7 +98,7 @@ public synchronized void unmountForced() throws VolumeException {
 
 	@Override
 	public Optional<Path> getMountPoint() {
-		return Optional.ofNullable(mountPoint);
+		return Optional.ofNullable(mountPoint); //TODO
```

Comment:
TODO what?

Reply:
TODO another issue 😅 i'll remove it from the PR.